### ISSUE TITLE
Multiple possible checksums for a file isn't working

### ISSUE THREAD

**Issue by KrisShannon** [wp-cli/checksum-command#24]
The meta api allows for multiple checksums for a file by returning an array.  e.g. for [plugin wptouch version 4.3.22](https://downloads.wordpress.org/plugin/wptouch.4.3.22.zip) the [checksums](https://downloads.wordpress.org/plugin-checksums/wptouch/4.3.21.json) for the file `lang/wptouch.pot` are:

```json
{
  "md5": [
    "ee40a33bff8f84acd1d3497e7bc2f32c",
    "1ece63d7ab1970a349aeed4fb8737f9a"
  ],
  "sha256": [
    "dff12d0d154f72f1296f1ad614cc5c8cfd562ffe869e9d6171be95916838663c",
    "0db91fb078011c813b6e0122823a08625a58a0474450782be17ee99c00939aed"
  ]
}
```

I've got a POC patch which fixes `wp checksum plugin` - KrisShannon@5121fba7669e22ce7b9266cd0bfa8cefa76056a1 - but it doesn't include tests and I think maybe the "are these two equal strings or is this a string in this array" logic should be separated into it's own utility function.

**Comment by danielbachhuber** [comment]
Great find!

> I think maybe the "are these two equal strings or is this a string in this array" logic should be separated into it's own utility function.

Hm, not sure this is critical. Strict check with `in_array()` should be sufficient.

**Comment by schlessera** [comment]
@KrisShannon Would you like to provide a PR for this?

**Comment by schlessera** [comment]
Solved in #e3ad28a

Thanks for the report, @KrisShannon !
